testsuite/features/reposync/srv_add_rocky8_repositories.feature

```diff
@@ -1,119 +0,0 @@
-# Copyright (c) 2022 SUSE LLC
-# Licensed under the terms of the MIT license.
-
-@rhlike_minion
-Feature: Add the Rocky 8 distribution custom repositories
```

Comment:
Let's give it a try.
But I'm not fully sure if we might need to keep the CLM filters. And so to keep using the mounted DVD iso. Even if we don't synchronized anything via network.